torch/_dynamo/utils.py

```diff
@@ -135,10 +136,8 @@
 # profiling compilation time by function
 compilation_time_metrics: Dict[str, List[float]] = {}
 
-# profiling compilation time by frame phase
-frame_phase_timing: Dict[str, Dict[str, float]] = collections.defaultdict(
-    lambda: collections.defaultdict(float)
-)
+# cumulative times for any CompilationMetric field populated by dynamo_timed
```

Comment:
This is to support `calculate_time_spent()`

Reply:
Questions I will be asking: Does this eventually need to be thread local? Does this get clear()'ed when we exit all dynamo_timed spans?

Reply:
You should suffix this with `_us`